<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveTorrentDialog</class>
 <widget class="QDialog" name="SaveTorrentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="savePathLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeSavePathButton">
           <property name="text">
            <string>Change save dir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="informationBox">
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="sizeLabel">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sizeInfo">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="dateLabel">
              <property name="text">
               <string>Date:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateInfo">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="infoHashV1Label">
              <property name="text">
               <string>Info hash V1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoHashV1Value">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="infoHashV2Label">
              <property name="text">
               <string>Info hash V2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoHashV2Value">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="infoHashBestLabel">
              <property name="text">
               <string>Info hash best:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoHashBestValue">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="commentLabel">
              <property name="text">
               <string>Comment:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="commentValue">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="fileView"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveTorrentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveTorrentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
